cmd/kots/cli/set-config.go

```diff
@@ -132,13 +134,26 @@ func SetConfigCmd() *cobra.Command {
 			}
 
 			response := struct {
-				Error string `json:"error"`
+				Error            string                                        `json:"error"`
+				ValidationErrors []kotsadmvalidator.ConfigGroupValidationError `json:"validationErrors,omitempty"`
 			}{}
 			_ = json.Unmarshal(respBody, &response)
 
 			if resp.StatusCode != http.StatusOK {
 				if resp.StatusCode == http.StatusNotFound {
 					return errors.Errorf("app with slug %s not found", appSlug)
+				} else if resp.StatusCode == http.StatusBadRequest {
```

Comment:
this check is not necessary. can just check for the existence of validation errors.